SConstruct

```diff
@@ -17,7 +17,7 @@ def replace_flags(flags, replaces):
             flags[flags.index(k)] = v
 
 
-env = Environment()
+env = Environment(ENV = os.environ)
```

Comment:
I don't think this one is needed.

Reply:
> did you try it with homebrew

I don't have a mac.

> I couldn't build it cause it couldn't find my Cmake from homebrew which was in my $PATH

Yeah, but that should be addressed by the other 2 changes in `methods.py` (i.e. setting the `ENV = os.environ` for `ssl_env` and `rtc_env`).

Reply:
> did you try it with homebrew

So, I spent some time to trying out homebrew on a friend's mac.
Homebrew installs a symbolic link to `/usr/local/bin/` [by default](https://brew.sh/#question), which was correctly picked up by the build system (and I confirmed the build was successful).
Homebrew seems to have an option `--no-binaries` which skips the symbolic link (off by default), though it didn't seem to really be effective for me.
Anyway, removing the symbolic link `/usr/local/bin/` then resulted in the build system not finding `cmake`, and setting the specific env path (e.g. `export PATH=/usr/local/Cellar/cmake/3.25.0/bin:$PATH`) did not work, but as expected, changing **only** the line `rtc_env = Environment(ENV=os.environ)` was enough to get a working build.
So, after all, I think it might be better to just add an option to specify the cmake binary for such a cases, e.g. `scons cmake=/path/to/cmake`.

Reply:
Everyone on Mac uses home 
brew. This is literally only Scons project that doesn't just build. I don't think requiring people to add location of Cmake makes sense. Build is also broken on other platforms like Windows for many people, I had some similar issues

Reply:
> Everyone on Mac uses home
> brew. This is literally only Scons project that doesn't just build.

Again, the default hombrew installs a link in `/usr/local/bin`, so by default, it works.
Anyway, I spent way to much time arguing on this, I had to bother a friend with a mac to confirm that `rtc_env = Environment(ENV=os.environ)` was enough and the other 2 changes where not needed because you refused to try my requested changes.
I do not intend to waste more time discussing this.

Reply:
Your original comment read as tho  manually setting the cmake path, generally speaking not everything symlinks into /usr/local/bin with homebrew. 

I will try out your rtc_env = Environment(ENV = os.environ) change, I literally didn't see it.  and the code change seems to be made after this comment